02-run-tests/06-cmr-netcdf.ipynb

```diff
@@ -61,11 +61,11 @@
       "application/vnd.holoviews_exec.v0+json": "",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def load_earthaccess_data(concept_id, cmr_query):
</pre><p>Do you think we should move these functions, all except maybe <code>run_cmr_tile_tests</code> to .py files to simplify this notebook? I could imagine actually that the first 4 lines of <code>run_cmr_tile_test</code> are a part of the CMRTileTest initialization, but maybe the <code>get_*</code> functions go in a different file as they are not specific to CMR.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/tile-benchmarking/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>